src/sentry/services/hybrid_cloud/log/impl.py

```diff
@@ -59,22 +59,24 @@ def find_last_log(
 
 class OutboxBackedLogService(LogService):
     def record_audit_log(self, *, event: AuditLogEvent) -> None:
-        RegionOutbox(
+        outbox = RegionOutbox(
             shard_scope=OutboxScope.AUDIT_LOG_SCOPE,
             shard_identifier=event.organization_id,
             category=OutboxCategory.AUDIT_LOG_EVENT,
             object_identifier=RegionOutbox.next_object_identifier(),
             payload=event.__dict__,
-        ).save()
+        )  # type: ignore
```

Comment:
I was stumped on a better way to fix this than `# type: ignore`. The type-checking message is

```
Incompatible type for "payload" of "RegionOutbox" (got "Dict[str, Any]", expected "Union[str, Combinable, None]")
```

which is frustrating, because the model in question declares

```python
    payload: models.Field[dict[str, Any], dict[str, Any]] = JSONField(null=True)
```

So passing `Dict[str, Any]` appears to be very much correct, and AFAICT it's saying `expected "Union[str, Combinable, None]"` as a result of some magical Django nonsense with how the model class gets translated into what an instance should look like.

Reply:
No change after merging, I'm afraid.

Reply:
weird... there's something super cursed about that class for some reason.  if I copy the file to a different name the bug goes away 🤔